specs/merge/client_settings.md

```diff
@@ -0,0 +1,17 @@
+# The Merge -- Client Settings
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+This document specifies configurable settings that merge clients are expected to ship with.
+
+### Override terminal total difficulty
+
+To coordinate changes to [`terminal_total_difficulty`](fork-choice.md#transitionstore), clients
+should have a setting `--terminal-total-difficulty-override`.
+
+If `TransitionStore` has already been initialized, this just changes the value of
+`TransitionStore.terminal_total_difficulty`, otherwise it initializes `TransitionStore` with the specified
```

Comment:
```suggestion
`TransitionStore.terminal_total_difficulty`, otherwise this setting initializes `TransitionStore` with the specified, bypassing `compute_terminal_total_difficulty` and the use of an `anchor_pow_block`.
```